### ISSUE TITLE
Error upgrading a new database

### ISSUE THREAD

**Issue by emanueldima** [EUDAT-B2SHARE/b2share#1533]
I'm testing the upgrade process starting from a new database, created from scratch on current master (`5bb77a885`). When I run `b2share upgrade run` the following error is raised:

````
b2share upgrade run
/Users/edima/.virtualenvs/b2share-evolution/lib/python3.6/site-packages/dojson/contrib/to_marc21/model.py:22: UserWarning: MARC21 undo feature is experimental
  warnings.warn('MARC21 undo feature is experimental')
Traceback (most recent call last):
....
  File "/Users/edima/.virtualenvs/b2share-evolution/src/b2share/b2share/modules/upgrade/cli.py", line 64, in run
    last_migration = all_migrations[-1]
IndexError: list index out of range
````

**Comment by emanueldima** [comment]
This happens when the db is created with `b2share db create` and not with `b2share upgrade run`, as it should. The solution is to replace the call to `db create` with the call to `upgrade run` in the b2share scripts.

**Comment by emanueldima** [comment]
related to https://github.com/EUDAT-B2SHARE/b2share/issues/1517

**Comment by nharraud** [comment]
Same issue spotted when one calls `b2share alembic upgrade` and then `b2share upgrade run`.
We will disable both CLI commands `b2share alembic upgrade` and `b2share db create` in order to avoid confusions and bugs. Both of those commands are now replaced by `b2share upgrade run`.

**Comment by emanueldima** [comment]
solved in 2.1.0
